src/rpc/rawtransaction.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_RPC_RAWTRANSACTION_H
+#define BITCOIN_RPC_RAWTRANSACTION_H
+
+class CBasicKeyStore;
```

Comment:
Unknown types below: `CMutableTransaction` and `UniValue`.

Reply:
Still not addressed in f5a3e0d4a3b4b359714b7d380d7784b7ca0524c0

Reply:
Done

Reply:
Hmm. It must have gotten lost somewhere.